### ISSUE TITLE
Fix build issues on maven 3.3.x

### ISSUE THREAD

**Issue by marekhorst** [openaire/iis#559]
We are struggling with build failures on maven `3.3.9` which were not present in `3.0.5`.

In particular `iis-metadataextraction` build fails with compilation error:
```
] COMPILATION ERROR :                                                                                                                                                                     
[INFO] -------------------------------------------------------------                                                                                                                            
[ERROR] /home/azio/dnet-iis-git/iis-wf/iis-wf-metadataextraction/src/main/java/eu/dnetlib/iis/wf/metadataextraction/AbstractMetadataExtractorMapper.java:[22,36] package com.itextpdf.text.exceptions does not exist                                                                                                                                                                            
[ERROR] /home/azio/dnet-iis-git/iis-wf/iis-wf-metadataextraction/src/main/java/eu/dnetlib/iis/wf/metadataextraction/AbstractMetadataExtractorMapper.java:[190,57] cannot find symbol
  symbol:   class InvalidPdfException
  location: class eu.dnetlib.iis.wf.metadataextraction.AbstractMetadataExtractorMapper<T>
[INFO] 2 errors 
```
it seems `itextpdf` dependency is not propagated as transitive one.

**Comment by marekhorst** [comment]
Apparently `cermine-impl` 1.8 release got all dependencies removed:

http://maven.ceon.pl/artifactory/simple/kdd-releases/pl/edu/icm/cermine/cermine-impl/1.8/cermine-impl-1.8.pom

they are present in `cermine-impl` 1.9 version:

http://maven.ceon.pl/artifactory/simple/kdd-releases/pl/edu/icm/cermine/cermine-impl/1.9/cermine-impl-1.9.pom

**Comment by marekhorst** [comment]
> Apparently cermine-impl 1.8 release got all dependencies removed:
> 
> http://maven.ceon.pl/artifactory/simple/kdd-releases/pl/edu/icm/cermine/cermine-impl/1.8/cermine-impl-1.8.pom

Released `cermine-impl` 1.8 `pom.xml` file:

http://maven.ceon.pl/artifactory/simple/kdd-releases/pl/edu/icm/cermine/cermine-impl/1.8/cermine-impl-1.8.pom

was manually fixed which solves build issues.
